uvicorn/supervisors/basereload.py

```diff
@@ -76,6 +79,10 @@ def restart(self) -> None:
     def shutdown(self) -> None:
         self.process.terminate()
         self.process.join()
+
+        for _socket in self.sockets:
+            _socket.close()
```

Comment:
Let's use the same notation as the `server.py`.

```suggestion
        for sock in self.sockets:
            sock.close()
```